crates/bitwarden-crypto/src/keys/device_key.rs

```diff
@@ -0,0 +1,126 @@
+use crate::{
+    error::Result, AsymmetricCryptoKey, AsymmetricEncString, EncString, KeyDecryptable,
+    KeyEncryptable, SymmetricCryptoKey, UserKey,
+};
+
+/// Device Key
+///
+/// Encrypts the DevicePrivateKey
+/// Allows the device to decrypt the UserKey, via the DevicePrivateKey.
+#[derive(Debug)]
+pub struct DeviceKey(SymmetricCryptoKey);
+
+#[derive(Debug)]
+pub struct CreateDeviceKey {
```

Comment:
nit: Not sure of the naming scheme here but at a glance this tells me these are the inputs needed to create a Device Key. Maybe this should be `CreatedDeviceKey`?

Reply:
Good point, I've renamed it to TrustDeviceResponse which is inline with other structs. We'll want to revisit this in the future.